packages/search/src/RocketSearchOption.js

```diff
@@ -103,7 +103,7 @@ export class RocketSearchOption extends LinkMixin(LionOption) {
 
   render() {
     return html`
-      <img class="icon" src=${getIcon(this.section)} />
+      <img class="icon" src=${getIcon(this.section)} alt=${this.section} />
```

Comment:
could this have `alt=""` or `role="presentation"`? Does the image bear any semantic meaning, or is it just decoration?

Reply:
the image indicates in which "section" of the page the result is coming from